app/utils/results.py

```diff
@@ -223,3 +223,90 @@ def add_ip_card(html_soup: BeautifulSoup, ip: str) -> BeautifulSoup:
         # Inserting the element
         ref_element.insert_before(ip_tag)
     return html_soup
+
+
+def check_currency(response: str):
+    """Check whether the results have currency conversion
+
+    Args:
+        response: Search query Result
+
+    Returns:
+        list
+
+    """
+    soup = BeautifulSoup(response, 'html.parser')
+    a = soup.find(class_='ZINbbc xpd O9g5cc uUPGi')
+    tag = a.find('a')
+    if tag and tag['href'] == 'https://g.co/gfd':
+        currency1 = soup.find(class_="xUrNXd UMOHqf")
+        currency2 = soup.select_one('.kCrYT'
+                                    ' .BNeawe.iBp4i.AP7Wnd '
+                                    '.BNeawe.iBp4i.AP7Wnd')
+        currency1 = currency1.text.rstrip('=').split(' ', 1)
+        currency2 = currency2.text.split(' ', 1)
+        return [float(currency1[0]), currency1[1],
+                float(currency2[0]), currency2[1]]
+    return None
+
+
+def add_currency_b(soup: BeautifulSoup,
+                   conversion_details: list) -> BeautifulSoup:
+    """Adds the currency conversion boxes
+    to response of the search query
+
+    Args:
+        soup: Parsed search result
+        conversion_details: list of currency
+        related information
+
+    Returns:
+        BeautifulSoup
+    """
+    # Element before which the code will be changed
+    # (This is the 'disclaimer' link)
+    element1 = soup.select_one('[class="nXE3Ob"]')
```

Comment:
I think this is fine since it's a single class, but it kinda makes me wonder if we should start maintaining an enum of classes used by Google, and then reference them using clearer names. Then it would also only require modifying one file if the class names ever change. So something like:

```py
class GClass(Enum):
    result_class = 'ZINbbc'
    currency_disclaimer = 'nXE30b'
    # etc
```

and then reference in these instances using something like:

```py
element1 = soup.select_one(f'[class="{GClass.currency_disclaimer}"]')
```

Don't worry about doing any of that in this PR, I can handle it later. I'm more just curious if you had any thoughts?

Reply:
Hey @benbusby. Sorry, I have been down with the flu and so didn't get time to check these out before 😅. I do think it would be a good idea to maintain an enum of classes used by google.

Reply:
Maybe after merging this pr, I could take up that issue. Since I am the one who has been using these classes a bunch!

Reply:
No worries! Hope you're feeling better!